### DISCUSSION TITLE
手机（ios12.5.5）无法正常加载文件

### DISCUSSION THREAD

**Discussion by H2Sxxa** [spencerwooo/onedrive-vercel-index#588] [Q&A] [answered]
![E36EB01B-BADF-45B2-B02E-3EFBBBC9191A](https://user-images.githubusercontent.com/88923783/161957873-84a5424e-172b-4930-ac9f-4d1cb95ead70.jpeg)
![9E485A5C-F328-48D5-AC03-B2098EC34B43](https://user-images.githubusercontent.com/88923783/161958389-eb0df858-6967-48a2-874a-cf55683ad124.jpeg)

音乐可能因为是flac无法播放？
不过pc倒是正常，没有出现以上的情况
会不会和浏览器内核有关？（瞎说的）

**Comment by H2Sxxa** [comment]
补充
文件夹可以访问
图片可以加载
wav也可以播放
问题可能只有readme.md

**Reply by H2Sxxa** [reply]
还有就是说这个问题不仅仅出现在我的页面上，访问其他人的网站也无法正常加载readme

**Reply by H2Sxxa** [reply]
又去试了一下
发现安卓的Chrome没有此问题
较新的iOS系统也无此问题
原因可能是safari浏览器内核版本不足
导致某些js无法加载，而出现了错误
蹲一波贡献者来验证猜想

**Comment by spencerwooo** [comment, marked as answer]
> 原因可能是 Safari 浏览器内核版本不足，导致某些 JavaScript 无法加载，而出现了错误。

应该是这样的，这个问题暂时放一放了，建议更新 iOS 15.
